hystrix/src/test/java/feign/hystrix/HystrixBuilderTest.java

```diff
@@ -148,7 +148,7 @@ public void fallbacksApplyOnError() {
   @Test
   public void errorInFallbackHasExpectedBehavior() {
     thrown.expect(HystrixRuntimeException.class);
-    thrown.expectMessage("contributors failed and fallback failed.");
+    thrown.expectMessage("feign#public abstract java.util.List feign.hystrix.HystrixBuilderTest$GitHub.contributors(java.lang.String,java.lang.String) failed and fallback failed.");
```

Comment:
Feign.configKey key is prettier :P I suspect the command key ends up in the UI somewhere, right? Might want to check that this doesn't make the operator experience ugly